File: app/src/main/java/com/unicorn/sxmobileoa/app/api/GeneralApi.java
package com.unicorn.sxmobileoa.app.api;

import com.unicorn.sxmobileoa.app.chongqing.Response;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface GeneralApi {

    @GET("request.shtml")
    Observable<Response> get(@Query("params") String params);

}

File: app/src/main/java/com/unicorn/sxmobileoa/app/api/GeneralApi.kt
package com.unicorn.sxmobileoa.app.api

import io.reactivex.Observable
import okhttp3.RequestBody
import okhttp3.ResponseBody
import retrofit2.http.Body
import retrofit2.http.Headers
import retrofit2.http.POST

interface GeneralApi {

    @Headers("Content-Type: text/xml")
    @POST("request.shtml")
    fun post(@Body requestBody: RequestBody): Observable<ResponseBody>

}